Atomic/util.py

```diff
@@ -474,3 +500,11 @@ def validate_manifest(spec, img_rootfs=None, img_tar=None, keywords=""):
     if keywords:
         cmd += ['-k',keywords]
     return subp(cmd)
+
+def get_signature_path(atomic_conf):
```

Comment:
Per https://github.com/projectatomic/atomic/pull/539#discussion_r75361262 :

---
We need to make this consistent with https://github.com/mtrmac/image/blob/docker-lookaside/docker/lookaside.go#L102 : 

 - Using URLs
 - `"registries".""` instead of `default-sigstore-path` (though that might be by changing `containers/image` instead)
 - `sigstore-write` values
 - No `/etc/pki/containers` default I think… A single sigstore for all of the internet really isn’t likely to be useful. For any external public repositories, there would be a read-only sigstore via HTTP. Read-write would be only the locally created/managed repositories (`mydepartment.mycompany.com/registry`) or something.

---
… and the per-registry/per-namespace/per-repository lookup from `lookaside.go` is necessary as well; we need various ISVs who publish to Docker hub to be able to maintain their own sigstore servers.